R/count_teleconnections.R

```diff
@@ -189,6 +195,21 @@ count_watershed_teleconnections <- function(data_dir,
           filter(GCAM_Class %in% crop_and_landcover_types$GCAM_Class) -> irr_crops
           length(irr_crops$irr_count) -> tc_n_irrigated_crops
 
+        # TELECONNECTION - Count # of economic sectors within watershed
+        # get raster values of land use types within the watershed.
+        if (city %in% c("New Orleans | LA", "Saint Louis | MO", "Laredo | TX", "Saginaw | MI", "Flint | MI", "Cleveland | OH")) {
```

Comment:
I think this list of cities appears a few times in the code... I'd suggest adding this as a variable in `Constants.R` (or could also be an argument of the count_teleconnections function). Name it something like `cities_for_raster_split` (you can probably come up with something snappier).